docs/endpoints/post-identity-map.md

```diff
@@ -19,7 +19,7 @@ Here's what you need to know:
 
 - The maximum request size is 1MB. 
 - To map a large number of email addresses, phone numbers, or their respective hashes, send them in *sequential* batches with a maximum batch size of 5,000 items per batch.
-- Unless you are using a Private Operator, do not send batches in parallel. In other words, use a single HTTP connection and map <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> consecutively.
+- Unless you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>, do not send batches in parallel. In other words, use a single HTTP connection and map <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> consecutively.
```

Comment:
i think the wording in docs/endpoints/post-optout-status.md is clearer and we can replicate here:

 _In other words, use a single HTTP connection and **send batches of** <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> consecutively._

Reply:
Done, thx.